api/v3/utils.php

```diff
@@ -2416,20 +2416,28 @@ function _civicrm_api3_api_resolve_alias($entity, $fieldName, $action = 'create'
  * @param string $entity
  * @param array $result
  *
- * @return string|array|null
+ * @return string|array|false
  */
 function _civicrm_api3_deprecation_check($entity, $result = []) {
-  if ($entity) {
+  if (!$entity) {
+    return FALSE;
+  }
+  $cacheKey = 'deprecation_check_entity_' . $entity;
+  if (!Civi::cache('metadata')->has($cacheKey)) {
     $apiFile = "api/v3/$entity.php";
```

Comment:
@colemanw as in just remove these lines?

Reply:
As in #16460